code/__DEFINES/power.dm

```diff
@@ -19,8 +19,12 @@
 ///The joule is the standard unit of energy for this codebase. You can use this with other defines to clarify that it will not be multiplied by time.
 #define JOULES * JOULE
 
-///The amount of energy, in joules, a standard powercell has.
-#define STANDARD_CELL_CHARGE (1 MEGA JOULES) // 1 MJ.
+///The capacity of a standard power cell
+#define STANDARD_CELL_VALUE (1 MEGA)
+	///The amount of energy, in joules, a standard powercell has.
+	#define STANDARD_CELL_CHARGE (STANDARD_CELL_VALUE JOULES) // 1 MJ.
+	///The amount of power, in watts, a standard powercell can give.
+	#define STANDARD_CELL_RATE (STANDARD_CELL_VALUE WATTS) // 1 MW.
```

Comment:
I think the documentation on this one could be improved. The description implies that there is some limit on the power usage of a cell, which is not the case. Also why is it indented?

Reply:
Indendation is simply for readibility. It shows us that this define depends on(or you could say a subtype) of the above define.

Here's another example 
https://github.com/tgstation/tgstation/blob/1c81752af4cfecce02114809895a3cf15b962ec3/code/__DEFINES/dcs/signals/signals_material_container.dm#L6-#L7

The return value is related to the signal so to show that relationship we indent it.

Also yes you are right about the documentation, I don't know how else to better write it so I'm open to suggestions